themes/gatsby-theme-blog/src/utils/global.css

```diff
@@ -1,3 +1,4 @@
+/* todo: are the language specific styles needed here? */
```

Comment:
@amberleyromo do you have any thoughts on the RTL styles in this file? I assume they're here for the translations in the Overreacted blog, but not sure if we do anything like this in our other starters

Reply:
> I assume they're here for the translations in the Overreacted blog

Indeed, indeed.

When I started this I intended to include the internationalization element. I think we can remove it for now. Would love to add this feature in later though.